cms/envs/aws.py

```diff
@@ -251,28 +251,6 @@
 STUDIO_NAME = ENV_TOKENS.get('STUDIO_NAME', 'edX Studio')
 STUDIO_SHORT_NAME = ENV_TOKENS.get('STUDIO_SHORT_NAME', 'Studio')
 
-# Modules having these categories would be excluded from progress calculations
-PROGRESS_DETACHED_APPS = ['group_project_v2']
-for app in PROGRESS_DETACHED_APPS:
-    try:
-        app_config = importlib.import_module('.app_config', app)
-    except ImportError:
-        continue
-
-    detached_module_categories = getattr(app_config, 'PROGRESS_DETACHED_CATEGORIES', [])
-    PROGRESS_DETACHED_CATEGORIES.extend(detached_module_categories)
-
-# Modules having these categories would be excluded from progress calculations
-PROGRESS_DETACHED_APPS = ['group_project_v2']
-for app in PROGRESS_DETACHED_APPS:
-    try:
-        app_config = importlib.import_module('.app_config', app)
-    except ImportError:
-        continue
-
-    detached_module_categories = getattr(app_config, 'PROGRESS_DETACHED_CATEGORIES', [])
-    PROGRESS_DETACHED_CATEGORIES.extend(detached_module_categories)
-
```

Comment:
The new completion API has a different mechanism where a block can self-declare if it should not be part of completion calculations.